Doc/howto/clinic.rst

```diff
@@ -1389,18 +1391,18 @@ Writing a custom return converter is much like writing
 a custom converter.  Except it's somewhat simpler, because return
 converters are themselves much simpler.
 
-Return converters must subclass ``CReturnConverter``.
+Return converters must subclass :py:class:`!CReturnConverter`.
 There are no examples yet of custom return converters,
 because they are not widely used yet.  If you wish to
-write your own return converter, please read ``Tools/clinic/clinic.py``,
-specifically the implementation of ``CReturnConverter`` and
+write your own return converter, please read :source:`Tools/clinic/clinic.py`,
+specifically the implementation of :py:class:`!CReturnConverter` and
 all its subclasses.
 
 
 How to convert ``METH_O`` and ``METH_NOARGS`` functions
 -------------------------------------------------------
 
-To convert a function using ``METH_O``, make sure the function's
+To convert a function using :c:data:`METH_O`, make sure the function's
```

Comment:
```suggestion
To convert a function using :c:data:`!METH_O`, make sure the function's
```

Reply:
See https://github.com/python/cpython/pull/106876/files#r1267881674. I think we should add the `METH_O` target first.

Reply:
Resolved with 8f239567112a90ede907e3131e95da89a229efbb